docs/pages/desktop-access/troubleshooting.mdx

```diff
@@ -174,3 +174,25 @@ issue, you can
 your own third party certificate. Note that Active Directory is
 [extremely picky](https://docs.microsoft.com/en-us/troubleshoot/windows-server/identity/enable-ldap-over-ssl-3rd-certification-authority#requirements-for-an-ldaps-certificate)
 so take care to generate your certificates correctly.
+
+### Teleport Certificate Authority not Compatible
```

Comment:
While headings should be sentence case rather than title case ([style guide](https://goteleport.com/docs/docs/best-practices/#style-guide)), they are already sentence case in this page, and "Not" is capitalized. I think we should capitalize "Not" here as well until we can standardize sentence-case headings in the docs.